test/data/blurb9/_api_config.yaml

```diff
@@ -2,11 +2,9 @@ resources:
   discovery:
     path: /
     description: Provides a directory of available resources
-    actions:
-      get:
+    methods:
+      GET:
         method: GET
```

Comment:
This is the config object used to populate the api.yaml template, so its contents don't matter now unless they affect the outcome (and it's a pain to change them right now)